### ISSUE TITLE
Allow to collapse the Incoming/Outgoing changes tree

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode#223708]
I would like to be able to collapse the tree with the Incoming and Outgoing changes in the SCM pane. It has many nodes and it distracts me from the changes view:

<img width="1339" alt="Screenshot 2024-07-25 at 16 38 36" src="https://github.com/user-attachments/assets/ba1ed3b0-a5c8-466f-af7f-af1fc29f21bc">

**Comment by KevinVandrielAtHarbor** [comment]
Yes this was sudden and confusing for me.

**Comment by itibbers** [comment]
+1

**Comment by XueMeijing** [comment]
Definitely

**Comment by XueMeijing** [comment]
I found that there is a checkbox in the settings which can disable the Incoming/Outgoing feature.

![image](https://github.com/user-attachments/assets/0af75b77-f911-4d33-9876-09767bbace20)

**Comment by thomas-lochner** [comment]
I know my memory is terrible, but I swear I used to be able to collapse this in a previous version. I recently updated and couldn't seem to do this anymore. Am I losing it? Was this recently changed? I disabled it as I would rather _not_ show it than show it, but I do like to be able to easily see the `incoming/outgoing` tree when I want to (with out having to remember that the setting exists).
